src/test/java/ch3/CarTest.java

```diff
@@ -3,30 +3,54 @@
 import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class CarTest {
 
     @Test
     void 차는_위치를_가진다() {
         Car car = new Car();
-        int result = car.getPosition();
+        int result = car.position();
         assertThat(result).isEqualTo(0);
     }
 
     @Test
     void 차는_4이상이면_전진한다() {
         Car car = new Car();
         car.move(4);
-        int actual = car.getPosition();
+        int actual = car.position();
         assertThat(actual).isEqualTo(1);
```

Comment:
```suggestion
        Car expected = new Car(1);
        assertThat(car).isEqualTo(expected);
```

값을 꺼내와서 비교기보다는 객체간 비교를 해보아도 좋을것 같네요!